helm-charts/0.24.5/charts/spire/templates/spire-agent-cluster-role-binding.yaml

```diff
@@ -0,0 +1,23 @@
+# /*
```

Comment:
Split these to individual files to make it easier to diff/merge with `helm-charts-hardened`-generated files.